sdk/instance/codec.go

```diff
@@ -0,0 +1,10 @@
+package instancesdk
+
+import (
+	"github.com/mesg-foundation/engine/codec"
+	"github.com/mesg-foundation/engine/protobuf/api"
+)
+
+func init() {
+	codec.Codec.RegisterConcrete(&api.ListInstanceRequest_Filter{}, "mesg.types.ListInstanceRequest_Filter ", nil)
```

Comment:
why do you need to register ListInstanceRequest_Filter to the codec? I tried without and its working well.